charts/kyverno/templates/cleanup-controller/deployment.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
   namespace: {{ template "kyverno.namespace" . }}
   labels:
     {{- include "kyverno.cleanup-controller.labels" . | nindent 4 }}
+  {{- with .Values.cleanupController.annotations }}
+  annotations: {{ tpl (toYaml .) $ | nindent 4 }}
```

Comment:
```suggestion
  annotations:
    {{- tpl (toYaml .) $ | nindent 4 }}
```

Reply:
(applies to other deployments as well)

Reply:
Would this applied to services as well ?
https://github.com/kyverno/kyverno/blob/main/charts/kyverno/templates/admission-controller/service.yaml#L9

Reply:
Is this part of your pr ?

Reply:
no, it is not

Reply:
Ok, changing the service one makes sense for consistency reasons. Up to you, i can do a follow up if you prefer.